src/graphql/helpers.ts

```diff
@@ -437,6 +443,7 @@ export function formatProposal(proposal) {
     domain: proposal.spaceDomain,
     verified: proposal.spaceVerified,
     turbo: proposal.spaceTurbo,
+    turboExpiration: proposal.turboExpiration,
```

Comment:
Weirdly i get different expiration with spaces query and proposals query

```Graphql 
{
  space(id:"thanku.eth"){
    id
    turbo
    turboExpiration
  }
  proposals(where:{space_in:["thanku.eth"]}){
    id
    space{
      id
        turbo
        turboExpiration 
    }
  }
}
```

Reply:
fixec